src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
```

Comment:
According to BEM naming convention, modifiers should be used with the base block or element class. So instead of just .stars--1, it should be .stars.stars--1 to indicate that it's a modifier of the .stars block. The same applies to other modifiers like .stars--2, .stars--3, etc.